sql/parser/builtins.go

```diff
@@ -83,15 +86,15 @@ func (b builtin) match(types typeList) bool {
 // functions.
 var builtins = map[string][]builtin{
 	// TODO(XisiHuang): support encoding, i.e., length(str, encoding).
```

Comment:
@XisiHuang I'm curious about the importance of supporting encodings other that UTF-8. Are non-unicode encodings still widely used? Would it be a hardship to require that strings are always UTF-8?